WCGitTagsPlugin/WCGitTagsPlugin.m

```diff
@@ -140,17 +151,18 @@ - (void)presentTagsModal:(id)sender {
         if (!self.tagsWindow) {
             [self loadInterface];
         }
-        
-        self.beingPresented = YES;
-        [[NSApp keyWindow] beginSheet:self.tagsWindow completionHandler:^(NSModalResponse returnCode) {
-            [self.tagsWindow orderOut:self];
-            self.repository = nil;
-            self.beingPresented = NO;
-        }];
-        
-        self.watchDog.gitDirectoryURL = self.repository.gitDirectoryURL;
-        [self.watchDog start];
-        [self syncTags];
+        if (self.tagsWindow) {
```

Comment:
i've added this check

if method loadInterface can't load correctly the nib file beingPresented Bool changed state